.github/workflows/PR_summary.yml

```diff
@@ -99,23 +99,30 @@ jobs:
       run: |
         cd pr-branch
         touch moved_without_deprecation.txt
+        touch extraneous_deprecated_module.txt
         git checkout ${{ github.base_ref }}
         while IFS= read -r file
         do
           if grep ^deprecated_module "${file}" ; then
-            printf 'info: removed file %s contains a deprecation\n' "${file}"
+            printf '\n⚠️ **warning**: removed file `%s` contains a module deprecation: please add this in a follow-up PR instead\n' "${file}" |
```

Comment:
Shouldn't shellcheck have complained here as well?

Reply:
Stupid question: why so? What defect should I fix? :-)

Reply:
Ah, after merging master it complains. I blindly silenced it --- please review carefully whether this is correct.

Reply:
My understanding is that shellcheck sees the backticks `` `...` `` and assumes that you want the shell to expand the inner command  (`` `cmd` `` is essentially a synonym for `$(cmd)`).

It then notices that the backticks are contained in single quotes, which interpret the backticks literally (which is actually what you intend here!) and warns you that the command will not expand.